packages/apps/human-app/frontend/src/modules/worker/jobs-discovery/components/oracles-table-desktop.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+  MaterialReactTable,
+  useMaterialReactTable,
+} from 'material-react-table';
+import { createTableDarkMode } from '@/shared/styles/create-table-dark-mode';
+import { useColorMode } from '@/shared/contexts/color-mode';
+import { useOraclesTableColumns } from '../hooks/use-oracles-table-columns';
+import { type Oracle } from '../hooks';
+
+interface OraclesTableDesktopProps {
+  isOraclesDataPending: boolean;
+  isOraclesDataError: boolean;
+  oraclesData: Oracle[] | undefined;
```

Comment:
Maybe `oraclesData: Oracle[]`? You already have `oracleData ?? []` in the parent component